### ISSUE TITLE
Document structure improvements

### ISSUE THREAD

**Issue by mgifford** [dwp/accessibility-manual#29]
This page - https://accessibility-manual.dwp.gov.uk/best-practice/document-structure - like the rest of the manual contains some really great stuff.  Lots to build on and learn from.

[Language](https://accessibility-manual.dwp.gov.uk/best-practice/document-structure#language) 

Could be improved by explaining about the language of parts. It is a bit technical here, and many WYSIWYG editors may not have an easy way to insert a language attribute into it.  We have built it into Drupal, but even there it isn't enabled by default, and not something that a generic editor can enable.  More examples here will be useful, as most folks won't know how to hard code the HTML for <span lang="fr">Je Ne Sais Quoi</span>.  This isn't something that people are trained to do in my experience:

> If you have more than one language in your document, for example you have an English document which contains a French quote, then you should make sure you set the language correctly for each part.

It is common to say, "You should try to write in plain English and for a reading age of 9." but if you put the line above into https://hemingwayapp.com/ it shows as Grade 14.  We all do this. Plain English is hard to write. It is important to have some automated process to check. 

I also like to remind folks that we're writing for real world situations.  People are reading work email after have a drink or two with friends at the pub, while on their cell phone.  We can't assume that people reading this content are at the top of their game and have already have their 2nd cup of coffee in the morning.  Real world, especially in a pandemic, is more complicated than that. 

It was unclear to me what type of documents you were writing..

The only office tool I know that has accessibility checkers built-in is the one from Microsoft, so you might as well say that. If government uses other software that has it, it would be good to include it.  

Also, on PDFs, can you be [clear and consistent and tell people not to](https://gds.blog.gov.uk/2018/07/16/why-gov-uk-content-should-be-published-in-html-and-not-pdf/)? We know they will do it anyways, but ya, a [warning like this one would be good](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/accessible-pdfs). 

Plus, why not link to the great work over here https://www.gov.uk/guidance/publishing-accessible-documents

**Comment by abbott567** [comment]
All valid points. Have made updates. Thanks!

**Comment by abbott567** [comment]
Updated in v1.1.0
